critical/back_end/s3_platform_infra.tf

```diff
@@ -12,6 +12,30 @@ resource "aws_s3_bucket" "platform_infra" {
     }
   }
 
+  lifecycle_rule {
+    id      = "expire_old_versions"
+    enabled = true
+
+    transition {
+      days          = 30
+      storage_class = "STANDARD_IA"
+    }
+
+    noncurrent_version_transition {
+      days          = 30
+      storage_class = "STANDARD_IA"
+    }
+
+    noncurrent_version_transition {
+      days          = 60
+      storage_class = "GLACIER"
+    }
+
+    noncurrent_version_expiration {
+      days = 90
+    }
+  }
```

Comment:
What benefits do we get by going through this lifecycle over just expiring the versions?

Reply:
Slightly lower costs, although tbh it's just copy/pasta from whatever example I got this from. I can remove it.

Reply:
🍝